pyproject.toml

```diff
@@ -123,3 +123,16 @@ target-version = ['py38']
 [tool.check-wheel-contents]
 # W009: Wheel contains multiple toplevel library entries
 ignore = "W009"
+
+[tool.ruff]
+src = ["pytest"]
+select = [
+    "I",  # isort
+]
+
+[tool.ruff.lint.isort]
+force-single-line = true
+known-local-folder = ["pytest", "_pytest"]
+known-third-party = ["test_source", "test_excinfo"]
```

Comment:
Curious why this is needed (I know it's copied from isort config but maybe it's no longer needed?)

Reply:
Indeed, removing create no diff.